storybook/src/components/Menu/Menu.stories.tsx

```diff
@@ -6,47 +6,92 @@
 import type { Meta, StoryObj } from '@storybook/react-vite'
 
 import {
-  BarChartIcon,
-  DocumentsIcon,
-  FolderIcon,
+  BarChartFillIcon,
+  DocumentsFillIcon,
+  FolderFillIcon,
   PieChartFillIcon,
-  PieChartIcon,
-  SettingsIcon,
+  SettingsFillIcon,
 } from '@amsterdam/design-system-react-icons'
 import * as Icons from '@amsterdam/design-system-react-icons'
 import { Menu } from '@amsterdam/design-system-react/src'
+import { useEffect, useRef } from 'react'
+import { useArgs } from 'storybook/preview-api'
 
 const menuItems = [
   {
     href: '#',
-    icon: <PieChartIcon />,
+    icon: <PieChartFillIcon />,
     text: 'Dashboard',
   },
   {
     href: '#',
-    icon: <FolderIcon />,
+    icon: <FolderFillIcon />,
     text: 'Projecten',
   },
   {
     href: '#',
-    icon: <DocumentsIcon />,
+    icon: <DocumentsFillIcon />,
     text: 'Rapportages',
   },
   {
     href: '#',
-    icon: <BarChartIcon />,
+    icon: <BarChartFillIcon />,
     text: 'Analyses',
   },
   {
     href: '#',
-    icon: <SettingsIcon />,
+    icon: <SettingsFillIcon />,
     text: 'Instellingen',
   },
 ]
 
+const MIN_WIDTH_REM = 72.5
+
+const withInWideWindowArg = (StoryFn: any) => {
+  const [, updateArgs] = useArgs()
+  const lastValueRef = useRef<boolean | null>(null)
+
+  useEffect(() => {
```

Comment:
Can't we reuse `useIsAfterBreakpoint` here?

Reply:
I've looked into this, Storybook returns an error when we use the hook here. I've slightly cleaned up this function, let's use it for now.